bigtable/spark/src/main/scala/example/Wordcount.scala

```diff
@@ -0,0 +1,59 @@
+package example
+
+import org.apache.hadoop.hbase.spark.datasources.HBaseTableCatalog
+
+import scala.util.Try
+
+object Wordcount extends App {
+
+  println("Starting up...")
+
+  import org.apache.spark.sql.SparkSession
+  val spark = SparkSession.builder().master("local[*]").getOrCreate()
+
+  println("Spark version: " + spark.version)
+
+  val table = Try(args(0)).getOrElse("wordcount")
+  val catalog =
+    s"""{
+       |"table":{"namespace":"default", "name":"$table", "tableCoder":"PrimitiveType"},
+       |"rowkey":"key",
+       |"columns":{
+       |"col0":{"cf":"rowkey", "col":"key", "type":"string"},
+       |"col1":{"cf":"cf1", "col":"col1", "type":"boolean"},
+       |"col2":{"cf":"cf2", "col":"col2", "type":"double"},
+       |"col3":{"cf":"cf3", "col":"col3", "type":"int"}
+       |}
+       |}""".stripMargin
+  val numRecords = Try(args(1).toInt).getOrElse(10)
+  println(s"Saving $numRecords records to $table")
+  import spark.implicits._
```

Comment:
Not sure if this is a scala thing, but why are the imports sprinkled throughout the file?
- HBaseTableCatalog is import for the entire file
- SparkSession is in the function scope
- spark implicits are in the middle

Can you group them? Maybe move the non-implicits to the file level and the implicits to the top of the function

Reply:
`import spark.implicits._` is a Scala thing and needs to be after `val spark` to import values from the `spark` object.

Other imports are close to the place they're needed (and would not be surprised if represent part of the Scala code style of mine).